applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_phi_pop_layers/README.md

```diff
@@ -0,0 +1,10 @@
+# Test K<sub>0</sub> procedure with horizontal layers, material property PHI with POP using layers
+
+**Author:** [Jonathan Nuttall](https://github.com/mcgicjn2)
+
+## Case Specification
+A test for the K<sub>0</sub> procedure process.  The plane strain model consists of three horizontal soil layers.  The phreatic line coincides with the top boundary of the top soil layer.  Each soil layer has a linear elastic constitutive law associated with it.
+
+K<sub>0</sub> is calculated using Phi and the POP method. This also corrects for V<sub>ur</sub>.
```

Comment:
Is there a way to use the Greek letter $\nu$ here?

Reply:
tried with formula